src/appleseed/renderer/modeling/bssrdf/sss.h

```diff
@@ -288,6 +292,73 @@ void compute_absorption_and_scattering_mfp(
     }
 }
 
+//
```

Comment:
Move declarations with references, comments and documentation to the declaration part at the beginning of the file.